<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>acEditMenu</class>
 <widget class="QMenu" name="acEditMenu">
  <property name="title">
   <string>&amp;Edit</string>
  </property>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="../../Resources/acIcons.qrc">
     <normaloff>:/Icons/undo</normaloff>:/Icons/undo</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="../../Resources/acIcons.qrc">
     <normaloff>:/Icons/redo</normaloff>:/Icons/redo</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <addaction name="actionUndo"/>
  <addaction name="actionRedo"/>
 </widget>
 <resources>
  <include location="../../Resources/acIcons.qrc"/>
 </resources>
 <connections/>
</ui>
